### DISCUSSION TITLE
[Tooltip] transitionName vs motion - How to get the same behavior?

### DISCUSSION THREAD

**Discussion by hbatalhaStch** [ant-design/ant-design#41820] [Q&A]
I have a `Popover` where the arrow is placed in the mouse position and I want to remove the transition when popover is becoming visible. I was able to remove the transition by setting `transitionName` to `''` but `transitionName` is deprecated and it's advised to use `motion` instead. I can't seem to get the same behavior when I use:
```
motion={{
    motionName: ''
}}
```
Playgound:
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/basic-antd-5-4-2-forked-5teooj?file=/demo.tsx)

I also tried getting playing around with the other `motion` props but I can't get the desired behavior.

The behavior I get (using `motion={{ motionName: ''}}`):
![CPT2304141202-786x233](https://user-images.githubusercontent.com/114578587/232050844-a7970a67-a220-4d0c-8b1a-898be60f9ff8.gif)

The behavior I want (using `transitionName=''`):
![CPT2304141201-786x233](https://user-images.githubusercontent.com/114578587/232050885-dc149f92-f27c-4ddf-b3fe-1f54a1e9b1ed.gif)

**Comment by zombieJ** [comment]
ConfigProvider will support disable motion in next minor version

**Reply by dperetti** [reply]
Facing the same issue. Using the deprecated transitionName='' works. Using the new motion properties doesn't (tried all motion properties). Using the ConfigProvider does not either. (ant v. 5.23.0)
